### PR TITLE
s/sass/scss/

### LINKED ISSUE #15: rename "sass" helper "scss"
because there is actually ZERO SASS support, only SCSS:

https://github.com/sass/libsass/issues/16

### PR DESCRIPTION
**Pull request by f1337**
Resolves #15. Also suppressed the `console.log` debug output, b/c it's making the h9 tests hard to read. :santa:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "panda-9000",
-  "version": "2.0.0-alpha-03",
+  "version": "2.0.0-alpha-04",
```
Comment (diminish7) [inline package.json]: We never published `alpha-03` did we? No need to bump the version.

File: src/task.coffee [hunk 2]
```diff
@@ -82,8 +82,8 @@ _run = async (ran, name, args) ->
       else
         yield f args...
 
-    finish = Date.now()
-    console.log "Task '#{name}' completed in #{finish - start}ms."
+    # finish = Date.now()
+    # console.log "Task '#{name}' completed in #{finish - start}ms."
```
Comment (diminish7) [inline src/task.coffee]: What do you think about pulling these commented lines out?

Comment (f1337) [inline src/task.coffee]: No objection at all, but wasn't sure if @dyoder was done [debugging w/e he was debugging there](https://github.com/pandastrike/panda-9000/commit/52ff4879e7040d49887d4747b05c68af4ab9434f), so I didn't want to tear down [Chesterton's Fence](https://en.wikipedia.org/wiki/Wikipedia:Chesterton%27s_fence).

Comment (dyoder) [inline src/task.coffee]: Actually, I don't think was a debugging statement, I think it was a desire to add task completion times. But for some reason I think I just moved them somewhere else and forgot to remove them here.

Comment (dyoder) [inline src/task.coffee]: Or are you suggesting we just get rid of them?

**Comment by benniemosher** [comment]
👍

**Pull request merged**